src/main/java/frc/robot/subsystems/pivot/PivotIOSim.java

```diff
@@ -35,14 +38,13 @@ public class PivotIOSim implements PivotIO {
   public void updateInputs(PivotIOInputs inputs) {
     pivotSim.update(0.02);
 
-    inputs.leaderPosition = (pivotSim.getAngleRads() / Math.PI * 2); // Rotations
-    inputs.leaderVelocity = (pivotSim.getVelocityRadPerSec() / Math.PI * 2); // Rotations/Sec
+    inputs.leaderPosition = (pivotSim.getAngleRads() / (Math.PI * 2)); // Rotations
```

Comment:
I'm pretty sure there's a built in way to calculate rotations somewhere in wpi? @Ishan1522 dyk what it is

Reply:
yea there is one for radians to rotations: https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/util/Units.html#radiansToRotations(double) 

I had to create one for meters to rotations a bit ago bc they didn't have one 4 that.